packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/Sequences.tsx

```diff
@@ -622,6 +634,22 @@ export function RecordTable_Sequences(
 
   if (filteredRows == null) return null;
 
+  const warningText =
+    filteredRows.length > MAX_SEQUENCES_FOR_TREE ||
+    filteredRows.length < MIN_SEQUENCES_FOR_TREE ? (
+      <span>
+        To see a phylogenetic tree please use a filter to display between{' '}
+        {MIN_SEQUENCES_FOR_TREE.toLocaleString()} and{' '}
+        {MAX_SEQUENCES_FOR_TREE.toLocaleString()} sequences
+      </span>
+    ) : filteredRows.length < sortedRows.length ? (
+      <span>
+        Note: The ortholog group's phylogeny has been pruned to display only the
+        currently filtered proteins. This may differ from a tree constructed{' '}
+        <i>de novo</i> using only these sequences.
+      </span>
+    ) : undefined;
+
```

Comment:
I have a concern about putting this new message in the same place as the other message. It gets lost. Wouldn't it make more sense to add this to the table description where we describe the ability to filter?

Reply:
You mean it gets lost because it's the same colour and position as the "too many proteins" message that was almost certainly there just before they filtered?

Reply:
I'm not going to tackle this tonight. I don't really see it as a problem, personally. But we can discuss tomorrow.

Reply:
> You mean it gets lost because it's the same colour and position as the "too many proteins" message that was almost certainly there just before they filtered?

Yes, that was my thinking. Let's leave it as-is and see if others raise it as an issue.